timescale-forge/maintenance.md

```diff
@@ -0,0 +1,52 @@
+# Maintenance
+On Timescale Forge, software updates are handled automatically by us, and you do
+not need to perform any actions.
+
+Non-critical software updates are applied during a maintenance window that you
+can define to suit your workload.  If we detect a security vulnerability that
+affects you, we might need to perform maintenance outside of the scheduled
+maintenance window.
+
+<highlight type="important">
+After a maintenance update, the DNS name remains the same, but the IP address
+it points to changes.
+</highlight>
+
+Upgrades are performed as zero-downtime upgrades during maintenance windows.
+This means that there is no outage of your services during the upgrade. However,
+all connections and transactions in progress during the upgrade are reset.
```

Comment:
Should we mention something like: "most database connection systems should auto-retry their connection to restore service connections after this reset."?

Reply:
Sure!